docs/RowOriented.md

```diff
@@ -68,6 +68,120 @@ using (var rowReader = ParquetFile.CreateRowReader<MyRow>("example.parquet"))
 }
 ```
 
+## Reading and writing custom types
+
+The row-oriented API supports reading and writing custom types by providing a `LogicalReadConverterFactory` or `LogicalWriteConverterFactory`.
+The `LogicalTypeFactory` is required if the custom type is used for creating the schema (when writing), or if accessing a `LogicalColumnReader`
```

Comment:
When using the row-oriented API, I believe the `LogicalTypeFactory` will always be required as you can't access the lower level column based API, so it might be simplest to just say both the converter factory and type factory are required without going into further detail here.

Reply:
Thank you @adamreeve! 😄